### ISSUE TITLE
Toolbar not fixed/sticky in the slideout

### ISSUE THREAD

**Issue by adrienpicard** [verbb/vizy#316]
### Describe the bug

I'm using the following Vizy config with the toolbarFixed flag on. It works as expected on an entry page, but if the entry opens in a slide-out (as part of a matrix field for example), then the toolbar isn't fixed at the top.

```
{
  "buttons": ["italic", "bold", "unordered-list", "link", "undo", "redo"],
  "toolbarFixed": true
}
```

### Steps to reproduce

1. Create a Vizy field using the configuration above.  
2. Open an entry with the field in a slide-out.

### Craft CMS version

5.4.8

### Plugin version

3.0.4

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/vizy:"dev-craft-5 as 3.0.4"`.

**Comment by adrienpicard** [comment]
Thank you!

**Comment by engram-design** [comment]
Fixed in [3.0.5](https://github.com/verbb/vizy/releases/tag/3.0.5)
